components/dropzone/stories/dropzone.stories.js

```diff
@@ -16,10 +16,20 @@ export default {
       },
       control: "boolean",
     },
+    isFilled: {
```

Comment:
I noticed looking at the storybook that if you are viewing the Default variant and toggle `Filled` to true the "content" goes away and we are just left with the dashed border. Same thing occurs with the Dragged variant. 

Would we want it to be that you can only toggle `Filled` if it is `dragged`? if so I did something similar with ColorArea with disabled and focused.

Reply:
Yes good call out! I'll make that update